.travis.yml

```
@@ -38,7 +38,8 @@ install:
 script:
 # PlatformIO build
 # TODO: build all envs once possible
-- pio run --environment tantalus_stage_1
+- pio run --environment TantalusStage1
+- pio run --environment TantalusStage2
```

```suggestion
- pio run
```

If we do this it'll build all "environments" so we wont have to keep adding onto this list. Will just have to comment out the co-pilot "environment" until it actually builds.